server/models/user-model.js

```diff
@@ -26,29 +26,12 @@ const userSchema = new Schema({
         },
         default: ''
     },
-    password: {
```

Comment:
Do we want to keep encryption of anything? If not we could also remove bcrypt from node packages